### PR TITLE
Add unlawful detainer defenses

### LINKED ISSUE #169: Sample Unlawful Detainer Answer
[unlawful detainer answer and affirmative defenses.pdf](https://github.com/nonprofittechy/docassemble-MOHUDEvictionProject/files/11409899/unlawful.detainer.answer.and.affirmative.defenses.pdf)

https://www.stlcitycircuitcourt.com/CircuitClerk/unlawful%20detainer%20answer%20and%20affirmative%20defenses.pdf

### LINKED ISSUE #350: When the only claim is for unlawful detainer, do not claim warranty of habitability or breach of quiet enjoyment
I'm assuming that bad conditions + breach quiet enjoyment are not defenses to breach of lease.

### LINKED ISSUE #353: Defense no landlord-tenant relationship feedback
User was confused about this: 

 14. “There is no landlord-tenant relationship” (?) between PL and DEF.

This is not the first user feedback we have received on this point. We need to think how to explain this to the tenant if it is included.

### LINKED ISSUE #354: Explain occupancy permit
Attached screenshot: “occupancy permit” could be more defined.

@tobyfey is this resolved by the terms?

MOTenant-housing code

### LINKED ISSUE #367: Defenses for nonpayment cases only
make the warranty of habitability and breach of quiet enjoyment defenses conditional on it being a nonpayment case

### PR DESCRIPTION
**Pull request by nonprofittechy**
Fix #169 #166 fix #354 

Fix #367 
Fix #350 
Fix #353

### CODE REVIEW

**Comment by nonprofittechy** [comment]
Maybe should do this as a squash and merge. it has several commits that don't make sense as standalone changes.

File: docassemble/MOHUDEvictionProject/data/questions/MOHUDEvictionProject.yml [hunk 1]
```diff
@@ -521,6 +504,51 @@ subquestion: |
...
+      Even if what the landlord says happened is true, I deny that the reason the landlord gave is a good enough reason
+      to evict me.
+      % else:
+      Even if what the landlord says happened is true, the tenant denies that the reason the landlord gave is a good enough reason
+      to evict the tenant.
+      % endif
+    field: defense_breach_not_material
+    datatype: yesnoradio
+---
 id: defenses explanation
```
Comment (miabonardi) [inline docassemble/MOHUDEvictionProject/data/questions/MOHUDEvictionProject.yml:552]: The first portion of this question appears to only be for rent and passion cases, but showed on UD path. Does it apply to both? Might just be a language change?

**Landlord failed to state a claim for rent and possession**
  A landlord must include specific things in the landlord's petition in order to be able to evict.  Your landlord failed to include the following things: 

Screen Shot 2023-09-22 at 8 29 23 AM

**Review by miabonardi — CHANGES_REQUESTED** [review]
Rent and possession language in `id: defenses explanation` and in the final output document. 

Some might be simple language changes, but others might require checking whether the defense applies for both types of evictions. 

Screen Shot 2023-09-22 at 8 36 22 AM

**Comment by miabonardi** [comment]
> Maybe should do this as a squash and merge. it has several commits that don't make sense as standalone changes.

Will leave it to you to merge after we discuss my review

**Pull request merged**